tests/helpers/index.js

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
Did this new file come out of the box with the Ember upgrade? That's really neat!